### PR TITLE
1271/basic net monitoring ios

### PR DESCRIPTION
**Pull request by sushi2k**
I also moved "Burp plugins to Process Non-HTTP Traffic" from Android to the basic network testing chapter as they are independent from iOS/Android.

### CODE REVIEW

File: Document/0x06d-Testing-Data-Storage.md [hunk 1]
```diff
@@ -728,19 +728,9 @@ $ ls -alh MobileSync
 ls: MobileSync: Operation not permitted
 ```
 
-This is not a permission issue of the backup folder, but a new feature in macOS Mojave. Allow the Terminal or iTerm or whatever your terminal software is full disk access, which is explained [here](http://osxdaily.com/2018/10/09/fix-operation-not-permitted-terminal-error-macos/ "Fix Terminal “Operation not permitted” Error in MacOS Mojave") in detail. 
+This is not a permission issue of the backup folder, but a new feature in macOS Mojave. Allow the Terminal or iTerm or whatever your terminal software is full disk access, which is explained [here](http://osxdaily.com/2018/10/09/fix-operation-not-permitted-terminal-error-macos/ "Fix Terminal “Operation not permitted” Error in MacOS Mojave") in detail.
```
Comment (TheDauntless) [inline Document/0x06d-Testing-Data-Storage.md]: > Allow the Terminal or iTerm or whatever your terminal software is full disk access, which is explained

To

> Solve this problem by granting full disk access to your terminal application by following the explanation on [OSXDaily](...)

**Comment by OMTGreleaser** [comment]
Results for commit f1aa54db948174eba8b3a0305ca6e889a902b522: Broken link result: [!] Error(s) found by the Links validator: 2 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        4 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x04f-Testing-Network-Communication.md [hunk 2]
```diff
@@ -18,6 +18,17 @@ To use the interception proxy, you'll need run it on your machine and configure
...
+### Burp plugins to Process Non-HTTP Traffic
+
+Interception proxies such as Burp and OWASP ZAP won't show non-HTTP traffic, because they aren't capable of decoding it properly by default. There are, however, Burp plugins available such as:
+
+- [Burp-non-HTTP-Extension](https://github.com/summitt/Burp-Non-HTTP-Extension) and
+- [Mitm-relay](https://github.com/jrmdev/mitm_relay).
+
+These plugins can visualize non-HTTP protocols and you will also be able to intercept and manipulate the traffic.
+
+Please note that this setup can become sometimes very tedious and is not as straightforward as testing HTTP.
```
[the remark this answers is not in the document]
Comment (sushi2k) [reply Document/0x04f-Testing-Network-Communication.md]: done

**Comment by OMTGreleaser** [comment]
Results for commit 345ef45754abb773a91cddbc2e8114c619c43d7e: Broken link result: [!] Error(s) found by the Links validator: 2 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        4 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 22b0a0573e996b90fd5b17547c947fdbc38cbb7f: Broken link result: [!] Error(s) found by the Links validator: 2 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        4 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**